config/kubernetes/sig-testing/teams.yaml

```diff
@@ -66,6 +66,7 @@ teams:
     - chaodaiG
     - chases2
     - cjwagner
+    - dims
```

Comment:
we can do this in a follow up but let's review this team, we've reviewed these in OWNERS before but clearly need to go back over them here

Reply:
I would also suggest, perhaps in another PR, that we should add @aojea and @pohly, our other SIG TLs

Reply:
Sounds good. let me close this for now